Rubberduck.VBEEditor/VbeRuntime/VbeNativeApiAccessor.cs

```diff
@@ -37,36 +38,30 @@ private static IVbeNativeApi InitializeRuntime()
                     return new VbeNativeApi7();
                 case DllVersion.Vbe6:
                     return new VbeNativeApi6();
+                case DllVersion.Vb98:
+                    return new Vb6NativeApi();
                 default:
                     return DetermineVersion();
             }
         }
 
         private static IVbeNativeApi DetermineVersion()
         {
-            IVbeNativeApi runtime;
-            try
-            {
-                runtime = new VbeNativeApi7();
-                runtime.GetTimer();
-                _version = DllVersion.Vbe7;
-            }
-            catch
+            foreach (var type in new[] {typeof(VbeNativeApi7), typeof(VbeNativeApi6), typeof(Vb6NativeApi)})
             {
                 try
                 {
-                    runtime = new VbeNativeApi6();
+                    var runtime = (IVbeNativeApi)Activator.CreateInstance(type);
                     runtime.GetTimer();
-                    _version = DllVersion.Vbe6;
+                    return runtime;
                 }
                 catch
                 {
-                    // we shouldn't be here.... Rubberduck is a VBA add-in, so how the heck could it have loaded without a VBE dll?!?
-                    throw new InvalidOperationException("Cannot execute DoEvents; the VBE dll could not be located.");
+                    // ignored
                 }
             }
-
-            return _version != DllVersion.Unknown ? runtime : null;
+            // we shouldn't be here.... Rubberduck is a VBA add-in, so how the heck could it have loaded without a VBE dll?!?
+            throw new InvalidOperationException("Cannot execute DoEvents; the VBE dll could not be located.");
```

Comment:
See, one bad thing about editing a hacked code made by a hack is that you inherit all the hack's silliness, including hard-coding a name of method that was no longer used. We are calling `Timer`, not `DoEvents` here. Really, I should have written it as `Cannot execute a method from VBE dll; the VBE dll could not be located.`.

Reply:
So when a hack edits a hack made by a hack...  NVM, I'll push a new comment over lunch.